main.cpp

```diff
@@ -0,0 +1,34 @@
+//
+// This is a test program fro profiling performance
+//
+
+#include <cstdio>
+#include <cstdlib>
+
+#include "_aicspylibczi/Reader.h"
+
+int
+main(void){
+    pylibczi::Reader czi(L"/Users/jamies/Data/s_1_t_10_c_3_z_1.czi");
```

Comment:
another overly specific path

Reply:
So main.cpp was written solely for profiling against larger files. Should I just remove it from the project or what do you think the right thing to do is?

Reply:
It's a fair point.  One alternative is passing that file path as a command line arg.

Reply:
if you make this path a parameter and just stash the path to your favorite file in a comment or something, then other users will have a chance to run this